onnx/defs/schema.h

```diff
@@ -1078,6 +1092,7 @@ class OpSchemaRegistry final : public ISchemaRegistry {
    */
   static OpName_Domain_Version_Schema_Map& GetMapWithoutEnsuringRegistration();
   static OpName_Domain_Version_Schema_Map& map();
+  static int loaded_schema_version;
```

Comment:
i wonder whether this is really needed... it seems like you are only using it to determine whether the debug check should be done or not right? can you reuse the max_version\target version or some other way?

Reply:
For ONNX, yes it is just for DEBUG mode now, but I think it would be useful in Runtime. You can check more details in my comment above.